src/OpenCensus.Abstractions/Utils/Collections.cs

```diff
@@ -0,0 +1,71 @@
+﻿// <copyright file="Collections.cs" company="OpenCensus Authors">
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of theLicense at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenCensus.Utils.Abstractions
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+
+    internal static class Collections
+    {
+        public static string ToString<TKey, TValue>(IDictionary<TKey, TValue> dict)
+        {
+            if (dict == null)
+            {
+                return "null";
+            }
+
+            StringBuilder sb = new StringBuilder();
+            foreach (var kvp in dict)
+            {
+                sb.Append(kvp.Key.ToString());
+                sb.Append("=");
+                sb.Append(kvp.Value.ToString());
+                sb.Append(" ");
+            }
+
+            return sb.ToString();
+        }
+
+        public static string ToString<T>(IList<T> list)
```

Comment:
Do we need this function at all?

list == null ? null : string.Join(" ", list)?